.github/workflows/build_and_test.yml

```diff
@@ -224,6 +227,20 @@ jobs:
           source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
           .ci/run_tests.sh
 
+    - name: Update current time
```

Comment:
Updating the `now_time` in all the jobs seem to be a bit redundant. Can we do better here? Are yaml anchors supported in github action?